zipline/algorithm.py

```diff
@@ -899,15 +973,23 @@ def order_target_value(self, sid, target,
         order. If the position does exist, this is equivalent to placing an
         order for the difference between the target value and the
         current value.
+        If the Asset being ordered is a Future, the 'target value' calculated
+        is actually the target exposure, as Futures have no 'value'.
         """
         last_price = self.trading_client.current_data[sid].price
+
+        value_multiplier = 1
```

Comment:
Looks like this logic is repeated in multiple order functions, maybe factor out a `value_multiplier` function?